### PR TITLE
Name/Misc cleanup.  Fixed protx_register_fund

### LINKED ISSUE #106: v0.16 v0.17 Cleanup - Names/Misc
Reviewing PRs #88 and #105, I noted the following files that need review/cleanup:

* contrib/init/raptoreumd.openrcconf - dash -> raptoreum
* doc/release-notes/raptoreum - review release notes - do we want these here?
* doc/release-process.md  - dash -> raptoreum, review
*  contrib/gitian-keys - do we want these here?
* src/qt/README.md - dash -> raptoreum
* src/rpc/rpcevo.cpp - Dash/masternode references need to be fixed
* src/rpc/mining.cpp - get/setgenerate removed?
* src/rpc/misc.cpp - review validate/ismine/iswatchonly - is this an issue?  Units in duffs, dash-cli reference
* src/script/ismine.cpp - HaveKeys logic changed - all must have keys now instead of "at least one"?
* src/util.cpp - ">>>>>" merge conflict remnant
* search for `loggibng.h` - rename/remove?
* src/validationinterface.cpp - smartnode reverted to masternode
* src/Makefile.am - BLS_LIBS - lbls-dash?

### PR DESCRIPTION
**Pull request by CodeIsTheKey**
Resolves #106 (leaves bls and gitian for later updates).

* Mostly name cleanup
* Corrected protx_register_fund (collateral amount, still needs testing).
* Fixed error in Makefile.am

### CODE REVIEW

File: src/rpc/rpcevo.cpp [hunk 1]
```diff
@@ -469,23 +469,22 @@ static std::string SignAndSendSpecialTx(const CMutableTransaction& tx, bool fSub
 void protx_register_fund_help(CWallet* const pwallet)
 {
     throw std::runtime_error(
-            "protx register_fund \"collateralAddress\" \"ipAndPort\" \"ownerAddress\" \"operatorPubKey\" \"votingAddress\" operatorReward \"payoutAddress\" ( \"fundAddress\" submit )\n"
-            "\nCreates, funds and sends a ProTx to the network. The resulting transaction will move 1000 Dash\n"
+            "protx register_fund \"collateralAddress\" \"collateralAmount\" \"ipAndPort\" \"ownerAddress\" \"operatorPubKey\" \"votingAddress\" operatorReward \"payoutAddress\" ( \"fundAddress\" submit )\n"
```
Comment (CodeIsTheKey) [inline src/rpc/rpcevo.cpp:472]: One of two real changes.

File: src/rpc/rpcevo.cpp [hunk 2]
```diff
@@ -620,9 +619,11 @@ UniValue protx_register(const JSONRPCRequest& request)
     if (isFundRegister) {
         CTxDestination collateralDest = DecodeDestination(request.params[paramIdx].get_str());
         if (!IsValidDestination(collateralDest)) {
-            throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, strprintf("invalid collaterall address: %s", request.params[paramIdx].get_str()));
+            throw JSONRPCError(RPC_INVALID_ADDRESS_OR_KEY, strprintf("invalid collateral address: %s", request.params[paramIdx].get_str()));
         }
         CScript collateralScript = GetScriptForDestination(collateralDest);
+        collateralAmount = ParseInt32V(request.params[paramIdx + 1], "collateralAmount") * COIN;
```
Comment (CodeIsTheKey) [inline src/rpc/rpcevo.cpp:625]: This was missing (and the argument count checks above)

File: src/Makefile.am [hunk 3]
```diff
@@ -201,7 +201,7 @@ BITCOIN_CORE_H = \
   llmq/quorums_signing.h \
   llmq/quorums_signing_shares.h \
   llmq/quorums_utils.h \
-  loggibng.h \
+  logging.h \
```
Comment (CodeIsTheKey) [inline src/Makefile.am:204]: Typo in Makefile.am

**Review by npq7721 — APPROVED** [review]
LGTM

**Pull request merged**